lib/protocol/http/url.rb

```diff
@@ -33,7 +33,7 @@ def self.escape(string, encoding = string.encoding)
 			# Unescapes a percent encoded string.
 			def self.unescape(string, encoding = string.encoding)
 				string.b.gsub(/%(\h\h)/) do |hex|
-					Integer(hex, 16).chr
+					Integer(hex[1..], 16).chr
```

Comment:
Ahh, thanks for this. What do you think about using `$1` here instead? It might be a little faster since we already have the sub-pattern match.